### DISCUSSION TITLE
Bug Report: boyfriend not found

### DISCUSSION THREAD

**Discussion by MarlowBl4ze** [KadeArchive/Kade-Engine#1590] [Q&A]
So i added a new boyfriend in character and stuff but when i select him it says 'null object found' and then crashes

**Comment by napstaa967** [comment]
what is on character.hx where you have your custom boyfriend

**Comment by MarlowBl4ze** [comment]
package;

import flixel.FlxG;
import flixel.FlxSprite;
import flixel.graphics.frames.FlxAtlasFrames;
import flixel.util.FlxTimer;

using StringTools;

class Boyfriend extends Character
{
	public var stunned:Bool = false;

	public function new(x:Float, y:Float, ?char:String = 'bf')
	{
		super(x, y, char, true);
	}

	override function update(elapsed:Float)
	{
		if (!debugMode)
		{
			if (animation.curAnim.name.startsWith('sing'))
			{
				holdTimer += elapsed;
			}
			else
				holdTimer = 0;

			if (animation.curAnim.name.endsWith('miss') && animation.curAnim.finished && !debugMode)
			{
				playAnim('idle', true, false, 10);
			}

			if (animation.curAnim.name == 'firstDeath' && animation.curAnim.finished)
			{
				playAnim('deathLoop');
			}
		}

		super.update(elapsed);
	}
}


i didn't edit that

**Comment by ACardboardBox1** [comment]
no not that, we mean the boyfriend code in character.hx lol
